sys/arm64/arm64/efirt_support.S

```diff
@@ -74,7 +74,12 @@ ENTRY(efi_rt_arch_call_nofault)
 	mov	x10, #20
 	sub	x8, x10, x8, lsl #2
 	add	c9, c9, x8, uxtx
+#ifdef __ARM_MORELLO_PURECAP_BENCHMARK_ABI
+	sub     x9, x9, #1
```

Comment:
Drop the + 1 from the adr instead. Though probably this should always do an integer branch, actually, since that's how the compiler implements jump tables for purecap code.

Reply:
Done